samples/bluetooth/central_past/src/main.c

```diff
@@ -293,7 +293,7 @@ void main(void)
 		sync_create_param.options = 0;
 		sync_create_param.sid = per_sid;
 		sync_create_param.skip = 0;
-		sync_create_param.timeout = 0xa;
+		sync_create_param.timeout = 0xaa;
```

Comment:
This is an improvement, but ideally the timeout should be a function of the periodic advertising interval (which is known). 

For another usecase I've implemented a function that takes a PA interval, and returns a (IMO) fitting sync timeout: https://github.com/zephyrproject-rtos/zephyr/blob/main/subsys/bluetooth/audio/bass.c#L349-L370

Doing a quick search for `sync_create_param.timeout = 0xa;` also shows that this issue may appear in more than just this sample, and could consider fixing them all

Reply:
I have increased timeout on all the samples. Didn't want to change any of the tests, as they must be working as they are currently, I think they seemed to have shorter intervals.

For these samples, I think a function to work out the timeout would be unnecessary.